### PR TITLE
add parameter DATA_SENDING_INTERVAL

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: src/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -145,11 +145,15 @@ export interface IEventMtxStatus {
...
  * Device send data periodically using this interval.
- * TODO:
+ * deviceParameters.DATA_SENDING_INTERVAL = `1`.
  */
-// interface IParameterDataSendingInterval {
-//     seconds: number
-// }
+interface IParameterDataSendingInterval {
+    /**
+     * Minimal interval for data sending from device in seconds.
```
Comment (DarkPark) [inline src/CommandBinaryBuffer.ts:152]:
```suggestion
     * Minimal interval for data sending from device (in seconds).
```

File: src/CommandBinaryBuffer.ts [hunk 2]
```diff
@@ -145,11 +145,15 @@ export interface IEventMtxStatus {
...
+     * Real value = value + pseudo-random value which not more than `255` * `4`.
```
Comment (DarkPark) [inline src/CommandBinaryBuffer.ts:153]:
```suggestion
     * Real value = value + pseudo-random value which is not more than `255` * `4`.
```

File: src/CommandBinaryBuffer.ts [hunk 3]
```diff
@@ -858,11 +868,31 @@ class CommandBinaryBuffer extends BinaryBuffer {
...
+        this.seek(this.offset + 2);
+
+        return {
+            value: this.getUint8() * DATA_SENDING_INTERVAL_SECONDS_COEFFICIENT
+        };
+    }
+
+    private setParameterDataSendingInterval ( parameter: IParameterDataSendingInterval ) {
+        // set 'reserved' parameter to 0
+        this.setUint16(0);
```
Comment (DarkPark) [inline src/CommandBinaryBuffer.ts:882]:
```suggestion
        this.seek(this.offset + 2);
```

should be quicker

File: src/commands/downlink/SetParameter.ts [hunk 4]
```diff
@@ -7,6 +7,50 @@ import * as deviceParameters from '../../constants/deviceParameters.js';
 const COMMAND_ID = 0x03;
 
 const examples: TCommandExampleList = [
+    {
+        name: 'set minimal interval for data sending to 1 hour',
+        parameters: {
+            id: deviceParameters.DATA_SENDING_INTERVAL,
+            data: {value: 3600}
+        },
+        hex: {
```
Comment (DarkPark) [inline src/commands/downlink/SetParameter.ts:16]: let's write it in one line

File: src/CommandBinaryBuffer.ts (hunk 1 above)
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts]: fixed

File: src/CommandBinaryBuffer.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts]: fixed

File: src/CommandBinaryBuffer.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/CommandBinaryBuffer.ts]: fixed

File: src/commands/downlink/SetParameter.ts (hunk 4 above)
Comment (DeadByElpy) [reply src/commands/downlink/SetParameter.ts]: fixed in all commands

**Review by DarkPark — APPROVED** [review]

**Pull request merged**